scripts/kibana_proxy.py

```diff
@@ -125,8 +120,8 @@ def print_instructions():
                 for f in as_completed(futures):
                     assert f.result() is None
         finally:
-            for container in containers:
-                container.kill()
+            for _container in containers:
```

Comment:
I changed this because PyCharm's linter complains on https://github.com/DataBiosphere/azul/pull/2394/files#diff-1b30fd6bc694a52a577ebd8dc17e3b0cb0f615d5234cade71fd708aa3d9bbaa2R100 about shadowed name `container`. Not sure if this is the best thing to rename it to.

Reply:
I don't get that warning on `develop`. Are you using the recommended pycharm settings?

Reply:
Never mind -- I was not using the project's recommended settings. I'll revert this change.